### ISSUE TITLE
'import type' errors from TypeScript in Deno 1.4.0

### ISSUE THREAD

**Issue by GGICE** [siokas/denomander#29]
The reason for the error can be seen here: https://stackoverflow.com/a/63887967

The error message is as follows.

```
error: TS1371 [ERROR]: This import is never used as a value and must use 'import type' because the 'importsNotUsedAsValues' is set to 'error'.
import { Denomander } from "./Denomander.ts";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/denomander@0.6.3/src/interfaces.ts:1:1

TS1371 [ERROR]: This import is never used as a value and must use 'import type' because the 'importsNotUsedAsValues' is set to 'error'.
import { Arguments } from "./Arguments.ts";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/denomander@0.6.3/src/Validator.ts:2:1

TS1371 [ERROR]: This import is never used as a value and must use 'import type' because the 'importsNotUsedAsValues' is set to 'error'.
import { Kernel } from "./Kernel.ts";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/denomander@0.6.3/src/Validator.ts:3:1

TS1371 [ERROR]: This import is never used as a value and must use 'import type' because the 'importsNotUsedAsValues' is set to 'error'.
import { Command } from "./Command.ts";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/denomander@0.6.3/src/Validator.ts:4:1

TS1371 [ERROR]: This import is never used as a value and must use 'import type' because the 'importsNotUsedAsValues' is set to 'error'.
import { ValidatorContract } from "./interfaces.ts";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/denomander@0.6.3/src/Validator.ts:5:1

TS1371 [ERROR]: This import is never used as a value and must use 'import type' because the 'importsNotUsedAsValues' is set to 'error'.
import { Option } from "./Option.ts";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/denomander@0.6.3/src/Validator.ts:13:1
```

**Comment by hkong** [comment]
This happened to me when I upgraded to Deno 1.4.2.  A temporary fix is to go back to Deno 1.3.3

**Comment by siokas** [comment]
update to the newest version of Deno
